cognite_toolkit/_cdf_tk/commands/clean.py

```diff
@@ -84,7 +84,7 @@ def clean_resources(
         # need to check for duplicates here as well.
         loaded_resources, duplicates = _remove_duplicates(loaded_resources, loader)
 
-        capabilities = loader.get_required_capability(loaded_resources)
+        capabilities = loader.get_required_capability(loaded_resources, False)
```

Comment:
When it is just a flag it is common to use a keyword argument. In addition, shouldn't this depend on whether you run in dry run mode?
```suggestion
        capabilities = loader.get_required_capability(loaded_resources, read_only=dry_run)

```